app/utils/client/lib/roomTypes.js

```diff
@@ -117,6 +124,18 @@ export const roomTypes = new class RocketChatRoomTypes extends RoomTypesCommon {
 		return this.roomTypes[roomType].notSubscribedTpl;
 	}
 
+	getReadOnlyTpl(roomId) {
+		const room = ChatRoom.findOne({ _id: roomId }, { fields: { t: 1 } });
```

Comment:
why not pass `t: { $exists: 1 }` if you agree plz check because there are more places like that